src/components/Calendar.js

```diff
@@ -133,7 +134,8 @@ Calendar.propTypes = {
   displayTime: bool,
   dayLabels: arrayOf(string),
   monthLabels: arrayOf(string),
-  timezone: string
+  timezone: string,
+  markedDays: arrayOf(number)
```

Comment:
I think that `markedDays` can also be a function (`oneOfType([arrayOf(number), func])`), like `disableDates` for more flexibility.